.buildkite/pipeline.yml

```diff
@@ -26,10 +26,10 @@ steps:
       # `build` task would also run tasks, which are executed in a separate job
       ./gradlew assemble
 
-  - label: "Unit tests"
+  - label: "Unit and screenshot tests"
     command: |
-      echo "--- 🧪 Testing"
-      ./gradlew test
+      echo "--- 🧪📸 Testing"
+      ./gradlew verifyRoborazziDebug
```

Comment:
All unit tests will be executed by the `verifyRoborazziDebug` task, not only screenshot ones.

Reply:
I tried to break one of the test in `AvatarUrlTest` and then I ran `./gradlew verifyRoborazziDebug`, I got a `BUILD SUCCESSFUL in 6s`, so I think our unit tests are not run by the `verifyRoborazziDebug` task.

Note that `AvatarUrlTest` and other unit tests are under the `gravatar` module, not `gravatar-ui`.

Reply:
One way to update it is to change this to `./gradlew :gravatar:test ` and add a new task to run `./gradlew :gravatar-ui:verifyRoborazziDebug`